modules/atomic-widgets/styles/transformers/color-transformer.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\Styles\Transformers;
+
+use Elementor\Modules\AtomicWidgets\Base\Style_Transformer_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Color_Transformer extends Style_Transformer_Base {
```

Comment:
You dont need this transformer, you can bind the type to `Primitive_Transformer`

Reply:
how? Don't forget we are still using the old style transformers system.

Reply:
Right sorry